pkg/skaffold/build/jib/sync_test.go

```diff
@@ -88,7 +88,7 @@ func TestGetSyncMapFromSystem(t *testing.T) {
 				dep1: SyncEntry{
 					[]string{dep1Target},
 					dep1Time,
-					false,
+					true,
```

Comment:
Why change that?

Reply:
just to introduce a failure so I could see if this works, it will be reverted.